### ISSUE TITLE
Update Deprecated Ubuntu Executor

### ISSUE THREAD

**Issue by findyourexit** [CircleCI-Public/azure-acr-orb#22]
### Orb version
0.2.0

### What happened
CircleCI have deprecated use of older Ubuntu executors, including `ubuntu-1604:201903-01` which is presently set as the default executor for this orb. Below image shows the message that appears in any CircleCI pipeline that makes use of `azure-acr-orb`:

<img width="782" alt="image" src="https://user-images.githubusercontent.com/762668/162097276-f7bf9169-bda6-40f4-a21c-bb27b3f3ebd9.png">

### Expected behavior
Migrate to a newer Ubuntu executor as per the above CircleCI notice, so that come late-May, the orb can continue to be used.

**Comment by AlaaNabil26** [comment]
I Have same issue , anyone has a solution for it ? or the orb wil be upgraded ?

**Comment by rodriguesk** [comment]
I have the same issue as well. I'm getting this for all my jobs because of this!
![Screen Shot 2022-06-01 at 4 52 11 PM](https://user-images.githubusercontent.com/19840874/171519977-81a897c8-4273-49ea-8eae-cf9707ebe8a4.png)

**Comment by findyourexit** [comment]
Can [this PR](https://github.com/CircleCI-Public/azure-acr-orb/pull/23) be expedited, as this orb is functionally broken now.

**Comment by HennaAbbas** [comment]
Hi @findyourexit, 

There are other issues we are looking to resolve before the PR can be merged. As a workaround, you can specify a image like in the example below: 

```
version: 2.1
orbs: 
  azure-acr: circleci/azure-acr@0.2.0

jobs:
   build: 
    machine: true
    steps:
    - azure-acr/build-and-push-image:    
        login-server-name: "my_server"
        registry-name: "registry_name"
        repo: "repo_name"

workflows:
  my_workflow:
    jobs:
      - build
```

**Comment by findyourexit** [comment]
> Hi @findyourexit @AlaaNabil26 @findyourexit , There are other issues we are looking to resolve before the PR can be merged. As a workaround, you can specify a image like in the example below:
> 
> ```
> version: 2.1
> orbs: 
>   azure-acr: circleci/azure-acr@0.2.0
> 
> jobs:
>    build: 
>     machine: true
>     steps:
>     - azure-acr/build-and-push-image:    
>         login-server-name: "my_server"
>         registry-name: "registry_name"
>         repo: "repo_name"
> 
> workflows:
>   my_workflow:
>     jobs:
>       - build
> ```

Unbelievable! Thank you very much for your assistance with the workaround @HennaAbbas, this worked a treat 💯

**Comment by EricRibeiro** [comment]
Merged in #23.
